nginx-nodejs-redis/nginx/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx
```

Comment:
Can you specify a tag for the base image please, we trying to advocate best practices and it's easier for us to know if our sample is using a compromised image